src/schema/me/__tests__/create_credit_card_mutation.test.js

```diff
@@ -0,0 +1,44 @@
+import { runAuthenticatedQuery } from "test/utils"
+
+describe("Credit card mutation", () => {
+  const creditCard = {
+    id: "foo-foo",
+    _id: "123",
+    name: "Foo User",
+    last_digits: "1234",
+    expiration_month: 3,
+    expiration_year: 2018,
+  }
+
+  const query = `
+  mutation {
+    createCreditCard(input: {token: "123abc"}) {
+      credit_card {
+        name
+        last_digits
+        expiration_month
+        expiration_year
+      }
+    }
+  }
+  `
+
+  const rootValue = {
+    createCreditCardLoader: sinon.stub().returns(Promise.resolve(creditCard)),
```

Comment:
Why not use Jest here?

Reply:
No reason

Reply:
This probably doesn't need to be a stub: `createCreditCardLoader: () => Promise.resolve(...)`

Reply:
Ah! You're right-- updated to jest and now updating to remove the stub. :)